docs/madr/decisions/038-meshmetric-profiles.md

```diff
@@ -0,0 +1,228 @@
+# MeshMetric profiles (predefined subsets of all metrics)
```

Comment:
we can approach profiles differently, one profile is a set of metrics, so we can combine profiles how we need

Reply:
https://github.com/kumahq/kuma/blob/master/docs/madr/decisions/017-policy-array-merging.md#pros-and-cons-of-the-options how to combine things ☝️